packages/styles/src/illustrated-message.scss

```diff
@@ -163,4 +163,43 @@ $block: fd-illustrated-message;
       }
     }
   }
+
+  &--dot {
+    padding: 0.25rem;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin: 0;
+
+    .#{$block}__illustration {
+      height: 2.813rem;
+      width: 2.813rem;
+      margin-right: 0.25rem;
+      &+.#{$block}__figcaption {
+        width: calc(100% - 2.813rem - 0.25rem);
+      }
+    }
+
+    .#{$block}__figcaption {
+      .#{$block}__title {
```

Comment:
do you need this specificity? Can't you just use `.#{$block}__title {...}`, `.#{$block}text {...}`?